caddyhttp/log/log.go

```diff
@@ -30,6 +31,19 @@ func init() {
 	})
 }
 
+type logReplacerKey struct{}
+
+var ctxKey logReplacerKey
```

Comment:
Can you change the name `ctxKey` here to something like logReplacerCtxKey to make it consistent with other context keys.

Reply:
actually I think we should create a const like the other context keys eg from middleware.go

    const (
         RemoteUserCtxKey caddy.CtxKey = "remote_user"

Reply:
@tobya thanks for the review!

I'm happy to make these these changes, but I'm curious what the utility of the shared `caddy.CtxKey` type is (I'm fairly new to the Caddy codebase). The utility I see in the private type is that it prevents any unintentional collisions between packages, which one can imagine resulting in confusing bugs in the general case.

Reply:
I amended my commit to use `logReplacerCtxKey` as the variable name. If you'd like me to make the switch to the shared context key type as well, just let me know, happy to adhere to the existing conventions.

Reply:
Thanks for making change. I’m fine with your explanation unless others disagree. We’ll
Need to do some more testing to make sure this change doesn’t introduce any unforeseen issues, but on the face of it this looks like a great solution